src/go/rpk/pkg/cli/cmd/cluster/partitions/status.go

```diff
@@ -0,0 +1,68 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewBalancerStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "balancer-status",
+		Short: "Queries cluster for partition balancer status",
+		Long: `Queries cluster for partition balancer status:
+
+FIELDS DESCRIPTION:
```

Comment:
Do we want to add more information on how to enable the auto balancer?

Reply:
Potentially worth a paragraph / sentence or two above this?

Reply:
+1

I'd potentially drop "DESCRIPTION", it might be implied -- what do we do in `topic produce`, `topic consume`, `acl` help?